packages/shared/src/components/tooltips/ChangelogTooltip.tsx

```diff
@@ -99,138 +90,122 @@ function ChangelogTooltip<TRef extends HTMLElement>({
+    !!post && (
+      <InteractivePopup
+        {...props}
+        position={InteractivePopupPosition.Unset}
+        className="bottom-0 left-60 mb-6 ml-6 border shadow-2 focus:outline-none w-[24rem] max-w-[360px] max-h-[466px] changelog bg-theme-bg-tertiary border-theme-color-cabbage"
+        data-testid="changelog"
+      >
+        <header className="flex flex-1 items-center py-3 px-4 border-b border-theme-divider-tertiary">
+          <h3
+            className="font-bold typo-title3 text-theme-label-primary"
+            data-testid="changelogNewReleaseTag"
+          >
+            New release
+          </h3>
+          <ModalClose
+            className="right-4"
+            onClick={onModalCloseClick}
+            data-testid="changelogModalClose"
+            buttonSize={ButtonSize.XSmall}
+          />
+        </header>
+        <section className="flex flex-col flex-1 p-5 h-full shrink max-h-full">
+          <Image
+            className="object-cover rounded-lg w-[207px] h-[108px]"
+            alt="Post cover image"
+            src={post.image}
+            fallbackSrc={cloudinary.post.imageCoverPlaceholder}
+            loading="lazy"
+            data-testid="changelogImage"
+          />
+          <h3
+            className="mt-3 font-normal text-theme-label-primary typo-title3"
+            data-testid="changelogTitle"
           >
-            <header className="flex flex-1 items-center py-3 px-4 border-b border-theme-divider-tertiary">
-              <h3
-                className="font-bold typo-title3 text-theme-label-primary"
-                data-testid="changelogNewReleaseTag"
-              >
-                New release
-              </h3>
-              <ModalClose
-                className="right-4"
-                onClick={onModalCloseClick}
-                data-testid="changelogModalClose"
-                buttonSize={ButtonSize.XSmall}
+            {post.title}
+          </h3>
+          <time
+            className="text-theme-label-quaternary typo-callout"
+            dateTime={post.createdAt}
+            data-testid="changelogDate"
+          >
+            {postDateFormat(post.createdAt)}
+          </time>
+          {!!post.summary && (
+            <div
+              className="mt-2 w text-theme-label-tertiary typo-callout"
+              data-testid="changelogSummary"
+            >
+              {post.summary}
+            </div>
+          )}
+          <div className="flex mt-4 font-bold w text-theme-color-salt typo-footnote">
+            <Button
+              id={`post-${post.id}-upvote-btn`}
+              icon={
+                <UpvoteIcon
+                  secondary={post?.userState?.vote === UserPostVote.Up}
+                />
+              }
+              pressed={post?.userState?.vote === UserPostVote.Up}
+              onClick={onToggleUpvote}
+              buttonSize={ButtonSize.Small}
+              className="btn-tertiary-avocado bg-theme-bg-tertiary"
```

Comment:
I think it's actually `quarternary` from design perspective, the bg should not be needed.

Reply:
So the quarternary button gives the green background on hover, while if I use button and add the bg theme, it removes that background. Thought that was needed from the design.
<img width="60" alt="Screenshot 2023-10-20 at 12 15 59 PM" src="https://github.com/dailydotdev/apps/assets/36197304/92f0bd92-5ae9-4bc0-8ba5-de438372eb76">
<img width="46" alt="Screenshot 2023-10-20 at 12 17 09 PM" src="https://github.com/dailydotdev/apps/assets/36197304/34008cac-da97-4093-a75a-cc6c55248826">

Is there any other way to make the quarternary button to not have that green background when hovered?

Reply:
Agreed to keep QuaternaryButton component to alight with the post cards buttons on the feed.

Reply:
Yep, just remove the background please here :)